.github/workflows/dockerbuild.yml

```diff
@@ -52,7 +53,7 @@ jobs:
           echo "Building $folder"
```

Comment:
wont SUBNAME=${NAMEFOLDER[0]} name the container image "edge-detect" instead of something like "edge-detect-yolo"?  Will one of the images overwrite the other during the CI process?